wiki/People/Tournament_Committee/es.md

```diff
@@ -1,10 +1,6 @@
----
-no_native_review: true
```

Comment:
not sure if you should remove that from the article you wrote yourself -- https://github.com/ppy/osu-wiki/pull/9624